docs/conf/portland/2020/hike.rst

```diff
@@ -12,14 +11,15 @@ It's rained on us the past, but we have faith it will be beautiful this year! We
 Logistics
 ---------
 
-**WARNING: This hike has been modified this year because of construction on the Wildwood trail. Instead of ending at the Zoo, we are doing an "out and back" hike to Pittock Mansion, which will start and end at the same location.**
+.. **You can get your hike ticket here**: FIXME
```

Comment:
We should probably just delete this sentence.

Reply:
We're gonna uncomment that later, :-D but sure.

Reply:
Oh, missed the comment tag.

Reply:
Oh, missed the comment tag.